src/object/obj_class.c

```diff
@@ -482,3 +496,64 @@ obj_ec_codec_get(daos_oclass_id_t oc_id)
 
 	return NULL;
 }
+
+
+/* Encode full stripe from SGL */
+int
+obj_encode_full_stripe(daos_obj_id_t oid, daos_sg_list_t *sgl, uint32_t *j,
+		       size_t *k, struct obj_ec_parity *parity, int p_idx)
+{
+	struct obj_ec_codec		*codec =
+				 obj_ec_codec_get(daos_obj_id2class(oid));
+ 	struct daos_oclass_attr 	*oca =
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs